build.gradle

```diff
@@ -367,7 +367,7 @@ task all(dependsOn: ['clean', 'build', 'slowtest', 'javadoc', 'allReports']) {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.2'
+    gradleVersion = '2.7'
```

Comment:
We already know that the build fails with 2.6 and greater of gradle, see the discussion on the developers list. This is something I am working on as it is non trivial.